RamilyaNigmatullina/pdp-2
# [#159311423] Add hotels

# Summary

Visitor should be able to see list of hotels

https://www.pivotaltracker.com/story/show/159311423

# Test plan

* Go to Application
* Open the home page
* See the list of hotels
